plugins/mips/mips_arithmetic.ml

```diff
@@ -0,0 +1,234 @@
+open Mips.Std
+
+(* ADD rd, rs, rt
+ * Add Word, MIPS32
+ * Page 36 *)
+let add cpu ops =
+  let rd = signed cpu.reg ops.(0) in
+  let rs = signed cpu.reg ops.(1) in
+  let rt = signed cpu.reg ops.(2) in
+  RTL.[ rd := rs + rt; ]
+
+(* ADD.S fd, fs, ft , MIPS32
+ * ADD.D fd, fs, ft , MIPS32
+ * ADD.PS fd, fs, ft , MIPS32, MIPS64
+ * Floating Point Add
+ * Page 37 *)
+let add_fmt cpu ops =
```

Comment:
I don't think that it's a good idea to add instruction that does nothing (if it isn't `nop` of course). `bap` doesn't support floating point right now, so there is definitely nothing to code in this case. But leave an empty RTL is not correct too because we code a wrong instruction semantic by doing this.  So I suggest to remove it. Or write here something like 
```
RTL.[
   message("add fmt not implemented")
]
```
But in the latter case, we probably should do it for all instructions we didn't implement. So do we really need this insn?